src/wasm/wasm-ir-builder.cpp

```diff
@@ -688,7 +702,15 @@ Result<> IRBuilder::makeCall(Name func, bool isReturn) {
   return Ok{};
 }
 
-// Result<> IRBuilder::makeCallIndirect() {}
```

Comment:
No need to implement this yet since it isn't used anywhere.

Reply:
done